### PR TITLE
Replaced customed GR API calls with a library: global-registry-nodejs-client

### PR DESCRIPTION
**Pull request by pkoz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,42 +23,43 @@
...
     "redis": "^2.8.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5",
-    "rollbar": "^2.4.6",
-    "sequelize": "^4.38.1",
+    "rollbar": "^2.5.0",
+    "sequelize": "^4.41.1",
     "sequelize-paper-trail": "^2.0.0",
-    "serverless-http": "^1.6.0",
+    "serverless-http": "^1.8.0",
```
Comment (mattdrees) [inline package.json]: lots of changes to dependency versions here that aren't really related to this commit. 
I'm not opposed to the changes, but could you put them in a separate commit please?

File: package-lock.json [hunk 2]
```diff
@@ -4418,6 +4418,13 @@
         "ini": "^1.3.4"
       }
     },
+    "global-registry-nodejs-client": {
+      "version": "github:CruGlobal/global-registry-nodejs-client#2d1d95e675432beaf5cc7c774617ffd1e9ae5e91",
+      "requires": {
+        "babel-eslint": "8.2.6",
```
Comment (mattdrees) [inline package-lock.json]: this reminds me --  https://github.com/CruGlobal/global-registry-nodejs-client/commit/af48b5a298c636b086a4183617d70d7dd6f7af99#r31231482

**Comment by mattdrees** [comment]
looking good!

File: package-lock.json (hunk 2 above)
Comment (pkoz) [reply package-lock.json]: Does not seem to be necessary, I have removed it from the source package

File: package.json (hunk 1 above)
Comment (mattdrees) [reply package.json]: looks like you decided to drop them from this PR. That's ok, as well.

**Comment by mattdrees** [comment]
@Omicron7 are you ok with us merging even with .003% decrease in test coverage? (Seems to be caused by the fact that Piotr reduced the total lines of code. His new code is covered just fine.)

**Comment by Omicron7** [comment]
👍 Still 100% coverage even with -0.003%

**Comment by mattdrees** [comment]
Brian looked over this and he's good with it. He'll (later on) move global-registry-nodejs-client to NPM under the CruGlobal organization.

I'll merge this. (There's not a great way to test this in stage, and it's not high-impact in prod, so we'll verify in prod.)

Piotr is tagging global-registry-nodejs-client as 1.0.0. I don't think we need to reference that tag from this project's package.json, though.

File: package-lock.json [hunk 3]
```diff
@@ -4418,6 +4418,12 @@
         "ini": "^1.3.4"
       }
     },
+    "global-registry-nodejs-client": {
+      "version": "github:CruGlobal/global-registry-nodejs-client#75beee7a2e7a63d1d1bac5218b033bae50596800",
```
Comment (mattdrees) [inline package-lock.json]: @pkoz I think it happened again. This commit isn't in that repo anymore. Maybe Travis didn't complain, because it was in that repo at the time of the build?

Anyway, can you make another small PR to fix this?
